rs-e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/rse2e/AutomatedTest.groovy

```diff
@@ -50,16 +52,19 @@ class AutomatedTest extends Specification  {
 
     def "test defined assertions on relevant messages"() {
         given:
+        def toRunRules = engine.getRules()
         def matchedFiles = fileMatcher.matchFiles(azureFiles, localFiles)
 
         when:
         for (messagePair in matchedFiles) {
             Message inputMessage = messagePair.getKey() as Message
             Message outputMessage = messagePair.getValue() as Message
-            engine.runRules(outputMessage, inputMessage)
+            def runRules = engine.runRules(outputMessage, inputMessage)
+            toRunRules.removeAll(runRules)
```

Comment:
I feel like the naming here might be confusing to come back to in the future since the `run` in `runRules` can either be a verb (like 'go run the rules') or an adjective ('run rules are rules that already ran'). Maybe they could be `evaluatedRules` and `rulesToEvaluate`, or `rulesThatRan` and `rulesToRun`, or some other pair that's slightly more distinct from the verb-based method names?

It might also be worth adding a small comment here, like `//Check whether all the rules in the assertions file have been run`

Reply:
Good point, I'll change the name